harness/determined/deploy/local/cli.py

```diff
@@ -159,13 +161,16 @@ def deploy_local(args: argparse.Namespace) -> None:
                     help="disable container auto-restart (recommended for local development)",
                     action="store_true",
                 ),
-                BoolOptArg(
+                Arg(
                     "--auto-bind-mount",
+                    type=str,
+                    default=None,
```

Comment:
I believe that should also default to `str(Path.home)`.

Reply:
Yeah, I handled this in `cluster_utils.py` logically but setting the default here would be cleaner. Will make a note to update this for the next notebooks PR. Good catch @ioga